lib/kb_phylogenomics/kb_phylogenomicsImpl.py

```diff
@@ -414,6 +399,8 @@ def trim_tree_to_genomeSet(self, ctx, params):
         # ctx is the context object
         # return variables are: output
         #BEGIN trim_tree_to_genomeSet
+        output = {}
+        raise NotImplementedError
         #END trim_tree_to_genomeSet
 
         # At some point might do deeper type checking...
```

Comment:
Why even keep this code around if you're thowing a NotImplementedError exception?

Reply:
@sychan, I think this is actually Dylan stubbing in future methods. I just wanted to add this code so my linter would shut up about "output" not exiting